.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: build
+
+on:
+  push:
+    paths:
+      - sdkexamples/**
+    branches:
+      - main
+  pull_request:
+    paths:
+      - sdkexamples/**
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: ['1.23']
```

Comment:
do we need to pin this, or can/should we default to "latest" ?

Reply:
https://github.com/helm/helm-www/pull/1677

Reply:
It contains these changes plus the requested changes

Reply:
I had one approval and I didn't want to reset that

Reply:
This looks great now 👍